managed/src/test/java/com/yugabyte/yw/controllers/CustomerTaskControllerTest.java

```diff
@@ -50,12 +54,18 @@
 import static play.test.Helpers.contentAsString;
 import static play.test.Helpers.fakeRequest;
 import static play.test.Helpers.route;
+import org.junit.runner.RunWith;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.mockito.InjectMocks;
 
+@RunWith(MockitoJUnitRunner.class)
 public class CustomerTaskControllerTest extends FakeDBApplication {
   private Customer customer;
   private Users user;
   private Universe universe;
 
+  @InjectMocks private CustomerTaskController controller;
```

Comment:
I sthis still needed? For commissioner status mocking?

Reply:
we do not need this, removed.